jupyter/minimal/ubi8-python-3.8/requirements.in

```diff
@@ -0,0 +1,2 @@
+jupyterlab==3.2.4
+
```

Comment:
Generated requirements.txt is missing, we want to commit it too.

Reply:
Ah okay, I thought that should be generated on the openshift-ci side